analyses/snv-callers/scripts/03-calculate_tmb.R

```diff
@@ -143,46 +140,109 @@ if (!dir.exists(opt$output)) {
   dir.create(opt$output, recursive = TRUE)
 }
 
-# Declare output file paths
-tmb_coding_file <- file.path(opt$output, "pbta-snv-consensus-mutation-tmb-coding.tsv")
-tmb_all_file <- file.path(opt$output, "pbta-snv-consensus-mutation-tmb-all.tsv")
+# Get data name
+data_name <- ifelse(opt$tcga, "tcga", "pbta")
+
+# Declare output file based on data_name
```

Comment:
I took out the word consensus here. Let me know if that is better or if there is another file name that would be more clear.